extensions/python/PythonScriptException.h

```diff
@@ -24,25 +24,12 @@
 #include <string>
 #include <stdexcept>
 
-namespace org {
-namespace apache {
-namespace nifi {
-namespace minifi {
-namespace script {
-/**
- * Defines an exception that may occur within the confines of a script.
- */
-class ScriptException : public std::exception {
+namespace org::apache::nifi::minifi::extensions::python {
+class PythonScriptException : public std::exception {
  public:
-  // Constructor
-  /*!
-   * Create a new exception
-   */
-  explicit ScriptException(std::string errorMsg)
-    : error_(std::move(errorMsg)) {
-  }
+  explicit PythonScriptException(std::string error) : error_(std::move(error)) {}
 
-  virtual ~ScriptException() noexcept = default;
+  virtual ~PythonScriptException() noexcept = default;
```

Comment:
similarly to `LuaScriptException` this could be problematic

Reply:
I've changed this aswell. https://github.com/apache/nifi-minifi-cpp/pull/1504/commits/4a89cb33d7af056a2ef1a14dacd6cb5cd1cde02f